src/pyk/testing/_kompiler.py

```diff
@@ -226,20 +232,32 @@ def kore_client(self, _kore_server: KoreServer, bug_report: BugReport) -> Iterat
             yield client
 
 
-class KCFGExploreTest(KoreClientTest, KPrintTest):
+class CTermSymbolicTest(KoreClientTest, KompiledTest):
```

Comment:
`KoreClientTest` extends `KompiledTest`:

```suggestion
class CTermSymbolicTest(KoreClientTest):
```

Reply:
Done.